src/main/java/io/jenkins/plugins/ml/utils/ConvertHelper.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * The MIT License
+ *
+ * Copyright 2020 Loghi Perinpanayagam.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package io.jenkins.plugins.ml.utils;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import hudson.FilePath;
+import org.apache.zeppelin.jupyter.JupyterUtil;
+import org.apache.zeppelin.jupyter.zformat.Note;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.Charset;
+
+
+public class ConvertHelper {
+
+    /**
+     * @param jupyterFile Path for the jupyter notebook
+     * @return Plain text which contains python code only
+     * @throws IOException          when file path does not exist
+     * @throws InterruptedException exception on input stream reading
+     */
+    public static String jupyterToText(FilePath jupyterFile) throws IOException, InterruptedException {
+        // Convert Jupyter Notebook to JSON
+        Note n = new JupyterUtil().getNote(new InputStreamReader(jupyterFile.read(), Charset.forName("UTF-8")), "%python", "\n", "#");
```

Comment:
See other comment similar to this one, about the resource not being closed, the line separator, etc.